app/models/person_upload.rb

```diff
@@ -0,0 +1,32 @@
+require 'virtus'
+class PersonUpload
```

Comment:
There isn't: although I wrote everything test-first, this didn't end up having any interesting behaviour that wasn't already exercised elsewhere - it doesn't have any branches, for example.